<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QInteractorStyleSurfaceCenterLineICDAStandard</class>
 <widget class="QWidget" name="QInteractorStyleSurfaceCenterLineICDAStandard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QInteractorStyleSurfaceCenterLineICDAStandard</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>ICDA diagnosis standard</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonCavernousSegmentOfLeftInternalCarotidArtery">
          <property name="text">
           <string>Left</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Cavernous segment of 
internal carotid artery:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxCavernousSegmentOfLeftInternalCarotidArtery">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string> mm</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButtonCavernousSegmentOfRightInternalCarotidArtery">
          <property name="text">
           <string>Right</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxCavernousSegmentOfRightInternalCarotidArtery">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="suffix">
           <string> mm</string>
          </property>
          <property name="maximum">
           <double>99999999999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonLeftMiddleCerebralArteryM1">
          <property name="text">
           <string>Left</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxLeftMiddleCerebralArteryM1">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string> mm</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Middle cerebral artery M1:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButtonRightMiddleCerebralArteryM1">
          <property name="text">
           <string>Right</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxRightMiddleCerebralArteryM1">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="suffix">
           <string> mm</string>
          </property>
          <property name="maximum">
           <double>99999999999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonLeftAnteriorCerebralArtery">
          <property name="text">
           <string>Left</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Anterior cerebral artery:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxLeftAnteriorCerebralArtery">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="suffix">
           <string> mm</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButtonRightAnteriorCerebralArtery">
          <property name="text">
           <string>Right</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxRightAnteriorCerebralArtery">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="suffix">
           <string> mm</string>
          </property>
          <property name="maximum">
           <double>99999999999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder></zorder>
     <zorder></zorder>
     <zorder></zorder>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>178</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
